### ISSUE TITLE
Draft for interview questions

### ISSUE THREAD

**Issue by andreuxxxx** [cncf/landscape#3096]
This is a space to write down any question that comes to mind that we may want to include in the interview plan. It's a brainstorm space to help us quick off the research plan

**Comment by Hilzygift** [comment]
Hi 
Here's a draft of the Research Interview guide and Research Questions 

**Research Interview Guide and Questions** 

**Introduction** 
Hello,
I am [Your name], a moderator conducting User experience research on behalf of the CNCF- Cloud Native Computing Foundation Landscape website. Our goal is to improve the user experience of the cloud computing landscape, and we are conducting interviews with stakeholders and users to gain insights and feedback.
We greatly value your input and would love to hear about the experiences you've had while using the CNCF landscape. Specifically, we are interested in learning about your needs, pain points, and any ideas you may have for improving the overall user experience.
The interview will take approximately [insert estimated time here]. Please let me know if you have any questions or concerns about the process.


**Warm-Up Questions** 

1. Can you tell me a little about yourself and your background?

**Main Research Questions** 

1. How did you first get interested in Cloud Network Computing Foundation?
2. What can you say is the main reason you use the CNCF Landscape?
3. Can you describe your experience with CNCF Landscape [product/service?
4. How often do you use CNCF Landscape?
5. Can you walk me through a recent experience you had with CNCF Landscape?
6. What are some challenges you’ve faced while using CNCF Landscape?
7. How important is the CNCF to your Work-related activities?
8. Have you used any similar products/services before? If so, can you compare your experiences?
9. How do you typically approach the information made available on the CNCF Landscape?
10. Can you describe your ideal (what you think it should be) experience with CNCF Landscape?

**Conclusion** 
Thank you so much for taking the time to speak with me today. Your feedback has been incredibly valuable, and it will help us improve our landscape website. Just to recap, we discussed your experience with the CNCF Landscape and what we can do to improve your experience. We will consider your feedback as we move forward with the project. If you have any additional feedback or questions, please don't hesitate to reach out. Once again, thank you for your time.

**Comment by nate-double-u** [comment]
> 7. How important is the CNCF to your Work-related activities?

Should this be: "7. How important is the CNCF landscape to your Work-related activities?"

**Comment by Hilzygift** [comment]
> > 7. How important is the CNCF to your Work-related activities?
> 
> Should this be: "7. How important is the CNCF landscape to your Work-related activities?"

Alright, I will effect the change
Thanks

**Comment by nate-double-u** [comment]
Thanks @Hilzygift, some great questions here. I've got some questions inline:

> Hi Here's a draft of the Stakeholder's Research Questions
> 
>     1. What inspired the creation of the CNCF Landscape?

The landscape is old, I think there may be one or two folks that could answer this. We may want to rephrase this to ask them what they think the landscape is for.
 
>     2. What are the main features of the CNCF Landscape?

Does this speak to the specific features that the stakeholders use, or that they are aware of?

>     3. What makes this landscape unique compared to others out there?
> 
>     4. What challenges does CNCF Landscape solve for the target audience?
> 
>     5. How would you describe the ideal user of CNCF Landscape?
> 
>     6. What are the current pain points or problems faced by the target audience that this CNCF Landscape solves?
> 
>     7. How does the landscape fit into the overall strategy of the company?

Foundation, not company 🙂 

>     8. What are the core values or principles that guide the development of CNCF Landscape?

This is an interesting question -- I think there are only a couple folks that could answer it in actuality, but I'd be interested in everyone's opinion on it 🙂
 
>     9. What kind of feedback have you received from users so far?

I'm not sure how to incorporate this into a question, but would this be both 'kind of feedback' ie, we've gotten positive feedback, and 'specific use feedback' ie, we hear that folks use the landscape in _this_ way to help them solve _this kind_ of problem?
 
>     10. What metrics are used to track the success of CNCF Landscape?

**Comment by Hilzygift** [comment]
Thank you for the feedback
> Thanks @Hilzygift, some great questions here. I've got some questions inline:
> 
> > Hi Here's a draft of the Stakeholder's Research Questions
> > ```
> > 1. What inspired the creation of the CNCF Landscape?
> > ```
> 
Alright, I think the CTO would best be part of those answering this particular question

> The landscape is old, I think there may be one or two folks that could answer this. We may want to rephrase this to ask them what they think the landscape is for.
> 
> > ```
> > 2. What are the main features of the CNCF Landscape?
> > ```
> 
I think the features they are aware of
> Does this speak to the specific features that the stakeholders use, or that they are aware of?
> 
> > ```
> > 3. What makes this landscape unique compared to others out there?
> > 
> > 4. What challenges does CNCF Landscape solve for the target audience?
> > 
> > 5. How would you describe the ideal user of CNCF Landscape?
> > 
> > 6. What are the current pain points or problems faced by the target audience that this CNCF Landscape solves?
> > 
> > 7. How does the landscape fit into the overall strategy of the company?
> > ```
> 
I will effect the change
> Foundation, not company 🙂
> 
> > ```
> > 8. What are the core values or principles that guide the development of CNCF Landscape?
> > ```
> 
Yeah 🤔
> This is an interesting question -- I think there are only a couple folks that could answer it in actuality, but I'd be interested in everyone's opinion on it 🙂
> 
> > ```
> > 9. What kind of feedback have you received from users so far?
> > ```
> 
I think both, or we option for feedbacks from the google analytics like @andreuxxxx mentioned in our last meeting
> I'm not sure how to incorporate this into a question, but would this be both 'kind of feedback' ie, we've gotten positive feedback, and 'specific use feedback' ie, we hear that folks use the landscape in _this_ way to help them solve _this kind_ of problem?
> 
> > ```
> > 10. What metrics are used to track the success of CNCF Landscape?
> > ```

**Comment by Hilzygift** [comment]
Hi @nate-double-u 

I had a meeting with @andreuxxxx and we decided to reduce the questions to 4 most important questions due to time constraint and to avoid lengthy interviews.

Here's the question;
1. What inspired the creation of the CNCF Landscape?
5. What challenges does CNCF Landscape solve for the target audience?
8. What are the core values or principles that guide the development of CNCF Landscape?
9. What kind of feedback have you received from users so far?

Your feedback would be much appreciated.

**Comment by Hilzygift** [comment]
**Research Interview Guide and Questions**  @andreuxxxx 

For Stakeholders: [Link](https://docs.google.com/document/d/1n0pgU-dzubL8H4FSpcNO_CyVFbwhUTnNaxKGTOR_PaE/edit?usp=sharing)
For Users: [Link](https://docs.google.com/document/d/1AL-xSZG0jgnOhA8HLvLySn5F5XPkKhh2fxingKLgNoo/edit?usp=sharing)

**Comment by tegioz** [comment]
Thanks @andreuxxxx @Hilzygift!

We've just migrated the [CNCF landscape to v2](https://landscape.cncf.io), so I'll close this one for now.

Please feel free to reopen if needed 🙂
